### ISSUE TITLE
Low contrast in Remote Explorer icons in Light High Contrast

### ISSUE THREAD

**Issue by alexdima** [microsoft/vscode#143743]
Testing #143425

<img width="316" alt="image" src="https://user-images.githubusercontent.com/5047891/155344665-bddd0641-f9ba-4b03-8af5-5e2805d4dc18.png">

**Comment by daviddossett** [comment]
Looks like another situation where we are applying opacity rules instead of actual colors.

https://github.com/microsoft/vscode/blob/e102191a6ffdc4bfc572c538913f292314092e48/src/vs/base/browser/ui/actionbar/actionbar.css#L54-L58

Same issue on dark HC theme:
<img width="300" alt="CleanShot 2022-02-23 at 10 56 46@2x" src="https://user-images.githubusercontent.com/25163139/155388308-90b227b2-69cf-461e-a8b0-8ecc95716578.png">

@isidorn thoughts on adding a disabled theme token? I could see this being at a slighter broader scope so it can be reused by other elements/areas in the UI.

**Comment by isidorn** [comment]
@daviddossett yes, in general I think it is ugly when we just apply opacity / transparency on top. All those should be treated as debt that we should fix imho.
I think this deserves a new colour.

Though let's hear what @aeschli thinks

**Comment by daviddossett** [comment]
I see a couple of options here:
1) We introduce a very general purpose token like `disabledForeground`. This gives other more specific tokens an easy way to inherit it.
2) We introduce something like `toolbar.disabledForeground` (and potentially other states if needed) to cover this specific situation.

We could still potentially use opacity in the token definition, but themes should be able to control that level rather than us hardcoding `0.4`.

Thoughts? cc @misolori

**Comment by misolori** [comment]
+1 on adding a new color token, but I believe the remote explorer is one of the areas that has yet to adopt the icon font so everything is in image backgrounds:

<img width="447" alt="CleanShot 2022-03-08 at 08 40 21@2x" src="https://user-images.githubusercontent.com/35271042/157283774-04db8271-dffd-4e72-bae0-0c20faebae2e.png">

I still think the new color token makes sense but the remote explorer icons appear to be a different problem

**Comment by daviddossett** [comment]
Got it, thanks for the heads up.

I'm leaning towards providing something top-level like `disabledForeground` so it can be picked up in other areas. I'll take a closer look at this next week.

**Comment by daviddossett** [comment]
I added `disabledForeground` at the very top level to enable more elements to take advantage a generic value. To start, I've used the new token in the action bar and the find widget. Both of which were using (different) opacity values for disabled states. They now use explicit colors e.g.

https://github.com/microsoft/vscode/blob/eec956770ef6bb6fe904460a912233b93accd609/src/vs/editor/contrib/find/browser/findWidget.css#L168-L171

## Action bar

<img width="267" alt="CleanShot 2022-03-30 at 10 45 57@2x" src="https://user-images.githubusercontent.com/25163139/160898560-e1a6eec9-1693-4ad5-be4c-09a0938a5e86.png">

<img width="256" alt="CleanShot 2022-03-30 at 10 45 43@2x" src="https://user-images.githubusercontent.com/25163139/160898571-97c16737-9556-4f6e-bd28-45c21bb13efb.png">

## Find widget

<img width="450" alt="CleanShot 2022-03-30 at 10 46 02@2x" src="https://user-images.githubusercontent.com/25163139/160898585-47e5aca4-522c-41d9-ad56-a595233aec00.png">

<img width="458" alt="CleanShot 2022-03-30 at 10 45 51@2x" src="https://user-images.githubusercontent.com/25163139/160898604-363a7ea6-29b5-4dd5-9d0b-c0e01dd2aef0.png">

**Comment by daviddossett** [comment]
Following up on this specific issue—I've raised a PR in the SSH repo to replace those assets. They will pick up this new disabled token once merged.
